demo/js/components/boot-nav.js

```diff
@@ -10,7 +10,9 @@ import RootPage from './RootPage';
  * @private
  */
 const pollForBrowserSync = callback => {
-  const bsScriptTag = document.getElementById('__bs_script__');
+  const bsScriptTag = Array.prototype.find.call(document.querySelectorAll('script'), elem =>
```

Comment:
Do we polyfill for `find` already?

Reply:
@joshblack Good question, and it's yes: https://github.com/IBM/carbon-components/blob/bed6f1eebb31e8a3e1e44cab0297c2c420a2dd09/demo/js/components/boot-nav.js#L1